react-viewer/src/App.tsx

```diff
@@ -1,11 +1,18 @@
 import "./App.scss";
 
-import { Viewer, IModelBackend } from "@itwin/web-viewer-react";
-import React from "react";
+import { Viewer } from "@itwin/web-viewer-react";
+import React, { useEffect, useState } from "react";
 
-import { NoSignInIAuthClient } from "./NoSignInIAuthClient";
+import { MyTokenServerAuthClient } from "./MyTokenServerAuthClient";
 
 const App: React.FC = () => {
+  const [isAuthorized, setIsAuthorized] = useState(
+    MyTokenServerAuthClient.oidcClient
+      ? MyTokenServerAuthClient.oidcClient.isAuthorized
+      : false
+  );
+  const [tokenUrl] = useState(process.env.TOKEN_URL ?? "http://localhost:3001/getToken" ); // defaults to the localhost version
```

Comment:
Nit: this doesn't necessarily need to be a state var like this because you're not ever setting it later, but you can leave this as is.